lib/rubocop/cop/metrics/block_length.rb

```diff
@@ -13,12 +13,28 @@ class BlockLength < Cop
         LABEL = 'Block'.freeze
 
         def on_block(node)
-          return if excluded_methods.include?(node.send_node.method_name.to_s)
+          return if excluded_method?(node)
           check_code_length(node)
         end
 
         private
 
+        def excluded_method?(node)
+          node_receiver = node.receiver && node.receiver.source.gsub(/\s+/, '')
```

Comment:
@pocke do you think it's ok to remove whitespaces from receiver?

I did it to cover this case:

```ruby
  Foo::
    Bar.baz
```

Reply:
Good idea 👍

Reply:
How about [`String#strip`](https://ruby-doc.org/core-2.2.0/String.html#method-i-strip) instead?

Reply:
@Drenmi, unfortunately, in our case the receiver contains whitespaces in the middle, i.e. in the aforementioned example, receiver source is `"Foo::\n  Bar"`.

But strip is a wonderful method, thanks!

Reply:
Noted. Makes sense. 🙂